### PR TITLE
Add additional payment types

### PR DESCRIPTION
**Pull request by admurray-fb**
# Purpose

We have multiple payment types that we support on the API - that are not present in the SDK this PR adds the missing payment types. 

# Related Material

See Issue https://github.com/freshbooks/freshbooks-nodejs-sdk/issues/105

### CODE REVIEW

**Review by seanmoon80 — APPROVED** [review]

File: packages/api/src/models/Payment.ts [hunk 1]
```diff
@@ -11,6 +11,20 @@ export enum PaymentType {
...
+	TwoCheckout = '2Checkout',
+	Visa = 'VISA',
+	Mastercard = 'MASTERCARD',
+	Discover = 'DISCOVER',
+	Nova = 'Nova',
+	Amex = 'AMEX',
+	Diners = 'DINERS',
+	Eurocard = 'EUROCARD',
+	Jcb = 'JCB',
+	Ach = 'ACH',
```
Comment (amcintosh) [inline packages/api/src/models/Payment.ts:27]: We also support "Other": https://github.com/freshbooks/evolve/blob/main/evolve/service/payment/payment_validate.py#L34

**Review by amcintosh — COMMENTED** [review]
We also need to support the new types in BillPayments here: https://github.com/freshbooks/freshbooks-nodejs-sdk/blob/master/packages/api/src/models/BillPayments.ts#L8

File: packages/api/src/models/BillPayments.ts [hunk 2]
```diff
@@ -9,6 +9,23 @@ enum PaymentType {
...
+	mastercard = 'MASTERCARD',
+	discover = 'DISCOVER',
+	nova = 'Nova',
+	amex = 'AMEX',
+	diners = 'DINERS',
+	eurocard = 'EUROCARD',
+	jcb = 'JCB',
+	ach = 'ACH',
+	googleCheckout = "Google Checkout",
+	amazon = "Amazon",
```
Comment (amcintosh) [inline packages/api/src/models/BillPayments.ts:27]: Hmm. Sorry to pester again, but I think the amazon and google ones are internal only as they don't seem to be in the frontend list: https://github.com/freshbooks/magnum-ui/blob/main/app/utils/payment-types.js

Comment (admurray-fb) [reply packages/api/src/models/BillPayments.ts:27]: np :) I'remove them.

**Review by amcintosh — APPROVED** [review]

**Pull request merged**